### ISSUE TITLE
Streamline find_quadicon in cfme/common/vm

### ISSUE THREAD

**Issue by mfalesni** [RedHatQE/cfme_tests#3040]
Currently it does too much of unnecessary navigation.

**Comment by mshriver** [comment]
Agreed, I overrode the function in cloud.instance, once infra.virtual_machine has navmazing it can be simplified. My changes to cloud.instance will be in PR 3738
